connectors/resteasy/pom.xml

```diff
@@ -19,6 +19,12 @@
 			<groupId>org.jboss.resteasy</groupId>
 			<artifactId>resteasy-jaxrs</artifactId>
 			<version>${resteasy-version}</version>
+			<exclusions>
+				<exclusion>
+					<artifactId>httpclient</artifactId>
+					<groupId>org.apache.httpcomponents</groupId>
+				</exclusion>
+			</exclusions>
```

Comment:
I used 3.1.3.Final. now I can see that they have released new version (3.1.4.Final).   I will try to use this version and will upgrade if it works. thanks